### ISSUE TITLE
Autocomplete window does not disappear if there are no suggestions/keeps capturing arrow keystrokes

### ISSUE THREAD

**Issue by bogger33** [texstudio-org/texstudio#1527]
### Environment
- TeXstudio: TeXstudio 2.12.14 (Build: 2.12.14+debian-1)
- Qt:  5.11.3, compiled with Qt 5.11.2 R
- OS: LMDE
- TeX distribution: pdflatex (the default)

### Expected behavior
If there are no suggestions, the autocomplete window disappears or at least stops capturing arrow key strokes, allowing navigation between lines (I understand keeping the window open is probably intentional, in case part of the command is erased again, but it could at least not block arrow keystrokes until this happens)

### Actual behavior
The autocomplete window stays behind, captung up/down keystrokes. Most of the time this is not a problem, but I've found this behaviour to really get in my way when I'm typing out algorithmic pseudocode algorithms. I would start by typing \IF{condition here}, make a couple of new lines, and type \ENDIF, so I don't forget later. After that, my natural instinct is to use the up key, to go back to the previous line and then insert the actual code that's supposed to go inside the if. However, nothing actually happens when I do that, because the autocomplete suggestion window captures this keystroke, so I'm forced to type a space after my \ENDIF before I can actually navigate up a line. 

### How to reproduce
paste in:

```
\begin{algorithm}
    \caption{}
    \begin{algorithmic}

    \end{algorithmic}
\end{algorithm}
```

type \ENDIF in the middle, try to press the up key on your keyboard to go up a line

**Comment by sunderme** [comment]
txs allows completion of algpseudocode. (not really sure what package you are actually using)
If you don't want to use the completer, close it with escape.
Auto-opening of completer can be disabled in option/completion.
algcompatible commands are added now, though txs 2.12.14 is quite old.

**Comment by bogger33** [comment]
I like the completer, I just don't like that it blocks me from using the up/down arrow when trying to navigate text, even though there's nothing to actually navigate up or down within the completer. Sure I can close it, but it always catches me by surprise when I do it, so it just slows things down a lot since I have to go "what? oh right" every time

**Comment by EasyArray** [comment]
I came here to post this, basically! For me, I'd like at least the *option* of having navigation keys (for me, `End`) close the completion menu, rather than navigating the menu itself. I know it only saves one keystroke (hitting `Esc`), but when I am typing `{\neq}` or `{\in}` or something short, I'd love to hit `End` to escape the menu and move past the closing `}` once I've typed it. But, as @bogger33 said, I do appreciate the completion menu for longer commands...
